packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/QrScannerModal.tsx

```diff
@@ -157,88 +128,51 @@ export const QrScannerModal = ({ onCancel, decision, allowPaste }: QrScannerModa
-                                { label: <Translation id="TR_QR_CODE" />, value: 'scan' },
-                            ]}
-                            selectedOption={isPasteMode ? 'paste' : 'scan'}
-                            onChange={value => setPasteMode(value === 'paste')}
-                        />
-                    )}
-                    {!isPasteMode && (
-                        <Description>
-                            <Translation id="TR_FOR_EASIER_AND_SAFER_INPUT" />
-                            <TrezorLink
-                                icon="externalLink"
-                                type="hint"
-                                href={HELP_CENTER_QR_CODE_URL}
-                            >
-                                <Translation id="TR_LEARN_MORE" />
-                            </TrezorLink>
-                        </Description>
-                    )}
+                    <Description>
+                        <Translation id="TR_FOR_EASIER_AND_SAFER_INPUT" />
+                        <TrezorLink icon="externalLink" type="hint" href={HELP_CENTER_QR_CODE_URL}>
+                            <Translation id="TR_LEARN_MORE" />
+                        </TrezorLink>
+                    </Description>
                 </DescriptionWrapper>
             }
-            bottomBarComponents={
-                isPasteMode ? (
-                    <ActionButton isDisabled={!text} onClick={() => handleScan(text)}>
-                        <Translation id="TR_CONFIRM" />
-                    </ActionButton>
-                ) : null
-            }
         >
-            {isPasteMode && (
-                <ContentWrapper $show>
-                    <StyledTextarea
-                        placeholder={`${translationString('TR_PASTE_URI')}…`}
-                        onChange={e => {
-                            setText(e.target.value);
-                        }}
-                    />
-                </ContentWrapper>
-            )}
-
-            {!isPasteMode && !readerLoaded && !error && (
-                <ContentWrapper $show>
-                    <CameraPlaceholder>
-                        <IconWrapper>
-                            <Icon name="qrCode" size={100} />
-                        </IconWrapper>
-                        <Translation id="TR_PLEASE_ALLOW_YOUR_CAMERA" />
-                    </CameraPlaceholder>
-                </ContentWrapper>
-            )}
-            {!isPasteMode && error && (
-                <ContentWrapper $show>
+            <ContentWrapper>
+                {error && (
                     <CameraPlaceholder>
                         <Error>
-                            <ErrorTitle>
+                            <Paragraph variant="destructive">
                                 <Translation id="TR_GENERIC_ERROR_TITLE" />
-                            </ErrorTitle>
+                            </Paragraph>
                             <ErrorMessage>{error}</ErrorMessage>
```

Comment:
Not 100% sure but I think it could be possible to replace also this one:

```
<Paragraph variant="destructive" align="center" margin={{ spacings.xxxl }}>...</Paragrap>
```

_Why I'm not sure: Because there is a span but at the same time i think shouldn't be there because we use align._